scripts/tag_version.py

```diff
@@ -0,0 +1,19 @@
+import os
+import sys
+import re
+
+def main(version):
+    os.system("git checkout main && git pull")
+    os.system("git tag -a v_%s -m 'Version %s'" % (version, version))
+    os.system("git push origin v_%s" % version)
+
+if __name__ == "__main__":
+    if len(sys.argv) < 2:
+        raise Exception("Please specify a version. Use format x.y.z.")
+    version = sys.argv[1]
+
+    if (not re.match(r'\d+\.\d+\.\d+', version)):
```

Comment:
I thought the version has "vX.Y.Z" format with "v" being required. Is this still the case?
If yes I would add it to the match pattern then drop it (will stabilise the regex a bit more).

Reply:
When calling the script you do not need to specify the `v`. The script will add it only where required